src/proposal.mligo

```diff
@@ -26,15 +26,15 @@ let check_if_proposal_exist (proposal_key, store : proposal_key * storage): prop
 
 // Gets the current period counting how many `voting_period` s have passed since
 // the 'started_on` timestamp. The periods start from zero index.
-let get_current_period_num(last_period_change, vp_length : last_period_change * nat) : nat =
+let get_current_period_num(last_period_change, vp_length : last_period_change * voting_period) : nat =
   match is_nat((Tezos.now - last_period_change.changed_on) : int) with
         | Some (elapsed_time) -> last_period_change.period_num + elapsed_time/vp_length
         | None -> ([%Michelson ({| { FAILWITH } |} : string * unit -> nat)]
             ("STARTED_ON_IN_FUTURE", ()))
```

Comment:
IIRC the need for the `last_period_change` was only to be able to track changes made using `set_voting_period`.

Now that we no longer have that entrypoint this can be simplified, we should no longer need to store the `period_num` of a `last_period_change` and so we can remove the latter entirely.
In the storage we can store a:
```ocaml
  ; start_time : timestamp
```
and here we can simplify the calculation as well, the period number should be the result of `elapsed_time/vp_length`

Reply:
Ah, yes. yesterday I reminded myself it should be removed, but missed it today. Will add now.

Reply:
Have replaced `last_period_change` from storage with only a timestamp and updated the calculations.